azure/application-gateway/main.tf

```
@@ -0,0 +1,279 @@
+resource "azurerm_public_ip" "main" {
+  count               = var.create_public_ip ? 1 : 0
+  name                = "pip-${var.name}"
+  resource_group_name = var.resource_group_name
+  location            = var.location
+  allocation_method   = "Static"
+  sku                 = "Standard"
+
+  tags = var.tags
+}
```

Update to ordering of properties

```suggestion
resource "azurerm_public_ip" "main" {
  count               = var.create_public_ip ? 1 : 0
  
  name                = "pip-${var.name}"
  resource_group_name = var.resource_group_name
  location            = var.location
  tags                = var.tags
  
  allocation_method   = "Static"
  sku                 = "Standard"
}
```